src/components/Header.js

```diff
@@ -0,0 +1,52 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+import './Components.css';
+import profileIcon from '../images/profileIcon.svg';
+import searchIcon from '../images/searchIcon.svg';
+import SearchBar from './SearchBar';
+
+function Header(props) {
+  const { title, search } = props;
+  const [hide, setHide] = useState(true);
+
+  const handleClick = () => {
+    if (hide === true) setHide(false);
+    if (hide === false) setHide(true);
```

Comment:
```suggestion
      hide ? setHide(false) : setHide(true);
```
Seria só uma sugestão que deixaria o código mais sucinto. Outra opção seria fazer o segundo `if` como else, já ele só pode ser `true` ou `false`